oauthlib/oauth1/rfc5849/endpoints/signature_only.py

```diff
@@ -34,13 +34,17 @@ def validate_request(self, uri, http_method='GET',
         """
         try:
             request = self._create_request(uri, http_method, body, headers)
-        except errors.OAuth1Error:
+        except errors.OAuth1Error as e:
```

Comment:
Nitpick: single-letter variable names are hard to grep for. Consider `err` or `error` instead?